### DISCUSSION TITLE
ix500 scan to network

### DISCUSSION THREAD

**Discussion by budda85** [jonaswinkler/paperless-ng#1163] [Support]
Does anyone here use an ix500 to scan directly to a network folder?
In the documentation it says that ftp and smb are possible but I can not find a suitable setting. 
Does anyone have a tip for me?

**Comment by budda85** [comment]
After my Internet search it is probably not possible to operate the scanner without a PC. 
According to the documentation, the S1300i can also use ftp and smb. But this has no wifi or lan. how does it work without a PC? 
Do I understand the table wrong? Are the connections only solvable with the ScanSnap software? It's best to write that with it or?

**Reply by mklik** [reply]
Same for me.
I also tried to scan directly to a network folder (consume) but find out, that this is only possible with a PC in the background. :-(
Would be happy to hear about any solution on that topic.

**Comment by kaiserbene** [comment]
ix500 does not have a network connection - so i used a raspberry pi with sane to provide that feature as the ix500 is a quite reliable scanner, provides two side scanning... even works with text recognition... using a 4B version to provide text recognition - as paperless-ng does this by itself an older version of raspberry should also do the trick.

**Reply by mklik** [reply]
The ix500 do have a WiFi connection!
So, to ease up things I thought I can directly scan to an network folder.
But interested in your solution with the Pi in detail and how do you start the scan itself. From the Pi or via the scan button of the ix500?

**Reply by kaiserbene** [reply]
used the blue button on the scanner.. had to delete the system because i needed the pi on another project... but it worked pretty well
